proximo-server/main.go

```diff
@@ -108,40 +109,34 @@ func main() {
 			if err != nil {
 				log.Panic(err)
 			}
-			cr.handler = nh
-			closer = nh
+			handler := nh
+			if err := serve(handler, counters, *port, *probePort); err != nil {
+				log.Fatal(err)
+			}
+			if err := nh.Close(); err != nil {
```

Comment:
I don't think this code would ever get executed. Consider putting this as `defer nh.Close()` before `serve()` and changing serve error handling as `log.Panic` instead of `log.Fatal`

Reply:
Why `Panic` instead of `Fatal`?

Also, I'm half tempted to just remove the `Close()` call.  Until we need a clean shutdown for some reason, it's not adding value.  Thoughts?

Reply:
`Panic`, calls `defer` statements, while `Fatal` doesn't.

We can remove `Close` for now and just add an issue for clean shutdown.